scripts/plotting.R

```diff
@@ -87,33 +82,41 @@ plot_points <- function(g, points_df) {
     mapping = aes(
       y = .data$value,
       group = interaction(.data$forecast_date, .data$forecaster)
-    )
+    ),
+    size = 0.125
   )
 
   return(g)
 }
 
-plot_state_forecasters <- function(
-    predictions_cards,
-    exclude_geos = c(),
-    start_day = NULL,
-    ncol = 5) {
-  if (nrow(predictions_cards) == 0) {
-    return(NULL)
-  }
+plot_state_forecasters <- function(predictions_cards, exclude_geos) {
+  assertthat::assert_that(nrow(predictions_cards) > 0)
+
+  td1 <- epidatr::pub_covidcast(
+    source = "hhs",
+    signals = "confirmed_admissions_covid_1d",
+    geo_type = "state",
+    time_type = "day",
+    geo_values = "*",
+    time_values = epidatr::epirange("2022-01-01", Sys.Date())
```

Comment:
This line means that any plots created after the original forecast date have data from the future. Unsure if feature or bug.
![image](https://github.com/cmu-delphi/exploration-tooling/assets/7095649/e10c6465-f9bf-4784-8321-30159e48330b)